src/cata_tiles.h

```diff
@@ -282,6 +282,7 @@ class cata_tiles
         int get_tile_width() const { return tile_width; }
         float get_tile_ratiox() const { return tile_ratiox; }
         float get_tile_ratioy() const { return tile_ratioy; }
+        tile_id_map get_tile_ids() const { return tile_ids; }
```

Comment:
> Languages that start with a P have spoiled me.

Yes, they are evil. Come and join the ~~dark~~, I mean the C++ side.

> Re the latter point, my use of tile_ids is not a check, I'm actually iterating through it, so no go on that.

Where do you think you iterate through it? The only time you use it is to call its `count` function.